agile-development/stand-ups/social-question/readme.md

```diff
@@ -0,0 +1,39 @@
+# Social question of the day
+
+The social question of the day is an optional short question to follow the [three project questions](../readme.md) in the daily stand-up. It develops team cohesion and interpersonal trust over the course of an engagement by facilitating the sharing of chosen personal context.
```

Comment:
Nit: "sharing of chosen personal context" consider replacing with "sharing of personal preferences, lifestyle or other context".